inspirehep/modules/theme/static/scss/format/brief-results.scss

```diff
@@ -252,21 +251,27 @@ article.search-result {
 
   .facet-title-truncate {
     width: 65% !important;
+    float:left;
     display: inline-block;
     white-space: nowrap;
     overflow: hidden;
     text-overflow: ellipsis;
+    padding-top: 3px;
   }
 
   label {
     width: 100%;
   }
+  .checkbox-container{
+  float: left;
```

Comment:
Small thing, but since the branch is out-of-date with current master, can you amend this indentation so that it is correct and rebase with `master` ?